packages/neos-ui-backend-connector/src/FetchWithErrorHandling/index.ts

```diff
@@ -95,7 +95,10 @@ class FetchWithErrorHandling {
                 if (response.ok) {
                     // CASE: all good; no errors!
                     resolve(response);
-                } else if (response.status === 401) {
+                } else if (
+                    response.status === 401 ||
+                    response.headers.get('X-Authentication-Required')
```

Comment:
Are you sure that this is always with upper-camel case? Believe me to remember that some browsers also use lower case, but I am not 100% sure. And HTTP header names are case-insensitive, according to RFC 2616.

Reply:
thanks for the hint! 
Foll disclosure: I'm not even 100% sure if this part is needed. This came in with a new Neos bugfix for Chrome 79 (see: https://github.com/neos/neos-development-collection/issues/2845)

The newest fix works also without this adjustment (the login prompt shows up), but just wanted to make sure to break nothing I do not understand / see. Maybe someone else has some insights.

Reply:
I'd keep that in place, and the case-sensitivity is no problem - otherwise that bugfix would not havwe worked ;)